servers/rendering/renderer_rd/forward_mobile/render_forward_mobile.cpp

```diff
@@ -686,10 +686,12 @@ void RenderForwardMobile::_pre_opaque_render(RenderDataRD *p_render_data) {
 		}
 
 		if (p_render_data->directional_shadows.size()) {
-			//open the pass for directional shadows
 			light_storage->update_directional_shadow_atlas();
-			RD::get_singleton()->draw_list_begin(light_storage->direction_shadow_get_fb(), RD::DRAW_CLEAR_DEPTH, Vector<Color>(), 0.0f);
-			RD::get_singleton()->draw_list_end();
+			if (!RenderingDeviceCommons::render_pass_opts_enabled) {
+				// Clear the shadow map now to overlap with some async compute.
+				RD::get_singleton()->draw_list_begin(light_storage->direction_shadow_get_fb(), RD::DRAW_CLEAR_DEPTH, Vector<Color>(), 0.0f);
+				RD::get_singleton()->draw_list_end();
+			}
```

Comment:
I would just remove this entirely. I see the point of having this to help with performance comparisons. But I would remove it before merging.

Reply:
This block is mostly to debug regression, because it allows us to toggle `render_pass_opts_enabled = false` and see if that immediately fixes the problem.

Ultimately the goal is to remove the render_pass_opts_enabled boolean once we are sure it's stable.